projects/ngx-datatable/src/lib/components/body/ghost-loader/ghost-loader.component.scss

```diff
@@ -0,0 +1,45 @@
+$datatable-ghost-loader-background-light: #f5f5f5 !default;
+$datatable-ghost-loader-background: #d9d8d9 !default;
+$datatable-ghost-loader-background-dark: #ccc7c7 !default;
+$ngx-datatable-default-background: #fff !default;
+$element-radius-0: 0 !default;
```

Comment:
Unfortunately, this does not work really. These vars get compiled into the component and are not present with the component shipped. Having the styles here is good, the variable part should go into the global theming. so that we can overwrite the vars when importing the theme.

One option I see is to use classes here for the variable parts and these are defined in the global styling.